content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -26,3 +51,37 @@ version. [Learn more about availability during upgrades here](../system-overview
     "Check Now" button; the new version should be recognized.
 1. Click "View Update".
 1. Review the release notes and then click "Install Update".
+
+### Replicated Command Line Interface
+
+1. Connect to the Terraform Enterprise host machine using SSH.
+1. Execute the command below and take note on the location from the property `AirgapPackagePath`:
+   
+    ```
+    $ replicatedctl params export | grep AirgapPackagePath
```

Comment:
@laurapacilio, on my second thought, perhaps we could change the command to this one which essentially making it self-contain

```suggestion
    $ replicatedctl params export --template '{{.AirgapPackagePath}}'
```